testing/vcs/test_boxfill_number_color_more_than_number_levels.py

```diff
@@ -34,6 +34,7 @@
     failed = True
 except:
     failed = False
+    raise
 
```

Comment:
good point.  I don't see the whole file right here but do we still need the line 39/40?